examples/basic/main.tf

```
@@ -0,0 +1,8 @@
+module "polygon-edge" {
+  source  = "aws-ia/polygon-technology-edge/aws"
+  version = "0.0.1"
```

so we dont have to update the example every new feature :D
```suggestion
  version = ">= 0.0.1"
```

---

Fixed in f190b110c8f5e63a56ae941ddfefec24e3d48fb1